app/init.php

```diff
@@ -60,6 +66,10 @@
 const APP_PAGING_LIMIT = 12;
 const APP_LIMIT_COUNT = 5000;
 const APP_LIMIT_USERS = 10000;
+const APP_LIMIT_ANTIVIRUS = 20000000; //20MB
+const APP_LIMIT_ENCRYPTION = 20000000; //20MB
+const APP_LIMIT_COMPRESSION = 20000000; //20MB
```

Comment:
Might be nice to split the number like you did for APP_STORAGE_READ_BUFFER

Reply:
answered above